keras_cv/layers/spatial_pyramid.py

```diff
@@ -31,22 +31,15 @@ class SpatialPyramidPooling(tf.keras.layers.Layer):
     inp = tf.keras.layers.Input((384, 384, 3))
     backbone = tf.keras.applications.EfficientNetB0(input_tensor=inp, include_top=False)
     layer_names = ['block2b_add', 'block3b_add', 'block5c_add', 'top_activation']
-
-    backbone_outputs = {}
-    for i, layer_name in enumerate(layer_names):
-        backbone_outputs[i+2] = backbone.get_layer(layer_name).output
-
-    # output_dict is a dict with 4 as keys, since it only process the level 4 backbone
-    # inputs
-    output_dict = keras_cv.layers.SpatialPyramidPooling(
-        level=4, dilation_rates=[6, 12, 18])(backbone_outputs)
+    backbone_outputs = backbone(inp)
```

Comment:
I didn't see any change on the resnet50 to let it produce backbone level output, eg calling model.as_backbone(). Do u want to update this example, or I missed anything.

Reply:
`backbone_outputs` here is a single Tensor. Renaming it to `output` intead.